core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3501,6 +3501,14 @@ public String toString() {
           .setScope(Scope.MASTER)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
+  public static final PropertyKey MASTER_METADATA_SYNC_USE_CLIENT_OPTION =
+      booleanBuilder(Name.MASTER_METADATA_SYNC_USE_CLIENT_OPTION)
```

Comment:
```suggestion
      booleanBuilder(Name.MASTER_METADATA_SYNC_IGNORE_TTL)
```

Reply:
Accept this suggestion with my commit [2fabfd4]